### PR TITLE
Upgrade zeebe to 8.1.1

### PR DESCRIPTION
**Pull request by arkadasfynarfin**
## Description

Config changes related to Zeebe upgrade

[GOV-449](https://mifosforge.jira.com/browse/GOV-449)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[GOV-449]: https://mifosforge.jira.com/browse/GOV-449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by fynmanoj** [comment]
Please rebase this PR @arkadasfynarfin

**Comment by fynmanoj** [comment]
@arkadasfynarfin update this 

Description
Describe the changes made and why they were made.
Add a link to teh design document or include the design bullet points related to this PR here.
(Ignore if these details are present on the associated JIRA ticket)

**Comment by arkadasfynarfin** [comment]
> @arkadasfynarfin update this
> 
> Description Describe the changes made and why they were made. Add a link to teh design document or include the design bullet points related to this PR here. (Ignore if these details are present on the associated JIRA ticket)

@fynmanoj This PR is rebased and description is updated with Jira ticket.

File: helm/ph-ee-engine/connector-ams-mifos/templates/integration-test.yaml
Comment (fynmanoj) [inline helm/ph-ee-engine/connector-ams-mifos/templates/integration-test.yaml:1]: this should be in a different chart.

**Pull request merged**